src/javascript/app/common/guide.js

```diff
@@ -38,18 +39,14 @@ const Guide = (() => {
         if ($(opt.guideBtnID).length === 0 || opt.script.length === 0) {
             return;
         }
-
+       
```

Comment:
same here

Reply:
Removed.

Reply:
I can still see the empty tabs and lines here

Reply:
I hope i removed it)